assets/src/components/routeVariantName.tsx

```diff
@@ -22,3 +22,26 @@ export const RouteVariantName = ({ vehicle }: { vehicle: VehicleOrGhost }) => {
     </div>
   )
 }
+
+export const RouteVariantName2 = ({ vehicle }: { vehicle: VehicleOrGhost }) => {
+  const route = useRoute(vehicle.routeId)
+  if (isVehicle(vehicle) && vehicle.isShuttle) {
+    return <div className="m-route-variant-name">Shuttle</div>
+  }
+
+  const { routeId, viaVariant, headsign } = vehicle
+  const viaVariantFormatted = viaVariant && viaVariant !== "_" ? viaVariant : ""
+
+  return (
+    <output title="Route Variant Name" className="m-route-variant-name">
```

Comment:
**question:** what's the rationale for using `output` here? I'm not the most familiar with it but from looking at the MDN docs I'm not really sure I see how it fits.

Reply:
`div` and `span` both are not semantically specific, where `output` is specifically for programmatic output from the web application, although I'm using `output` outside of html forms, I thought this was the most suitable element for conveying that data is outputted by the application, rather than static. 
As well as it made writing tests nicer because I could do `getByRole("status"`

While react does manage and efficiently update the DOM by diffing (although a implementation detail), I've been taking the approach I feel is cleaner and produces a better document by scoping changing or application produced values to an `output` element and leaving static text as apart of the document structure itself.

Reply:
Open to changing it, but I'm looking for a semantic element for the purpose of testing + accessibility that conveys properly that this is the output/rendering of data.

Reply:
Okay, makes sense! I think I have a tendency to interpret the meaning of various HTML elements a bit overly restrictively and reach for a `div` because something seems to be not _quite_ right about the more specialized elements I consider using.

Reply:
agreed!
On a similar note, I wanted to use a `<dl></dl>` instead of a table for key value pairs, but apparently even though `<dl>` is _the_ element for key value pairs essentially, `<table>` support is better on screen readers than for `<dl>`

(though, I don't know how many screen readers we're supporting if any)